mappings/net/minecraft/entity/EntityStatusTypes.mapping

```diff
@@ -0,0 +1,56 @@
+CLASS net/minecraft/class_6024 net/minecraft/entity/EntityStatusTypes
```

Comment:
Maybe just `EntityStatuses`? I don't see the need for a `Type` suffix.

Reply:
Yeah thats better. Ill fix it up one the byte or is merged

Reply:
Imo we can have this merged in first before the byte support is in. Consider changing the name now. If we delay merging, imo we are just getting more trouble as merge conflicts may come in over time.